content/en/docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch.md

```diff
@@ -330,7 +330,7 @@ With this patch, we indicate that we want to retain only the `type` key of the `
 Patch your Deployment again with this new patch:
 
 ```shell
-kubectl patch deployment retainkeys-demo --type merge --patch-file patch-file-retainkeys.yaml
+kubectl patch deployment retainkeys-demo --patch-file patch-file-retainkeys.yaml
```

Comment:
```suggestion
kubectl patch deployment retainkeys-demo --type strategic --patch-file patch-file-retainkeys.yaml
```

Although `strategic` is the default, making it explicit helps teach readers.

Reply:
updated